lib/extensions/postgres_cdc_rls/subscriptions.ex

```diff
@@ -36,26 +36,32 @@ defmodule Extensions.PostgresCdcRls.Subscriptions do
         subscription_id,
         entity,
         filters,
-        claims
+        claims,
+        action_filter
       )
       select
         $4::text::uuid,
         sub_tables.entity,
         $6,
-        $5
+        $5,
+        $7
       from
         sub_tables
         on conflict
-        (subscription_id, entity, filters)
+        (subscription_id, entity, filters, action_filter)
         do update set
         claims = excluded.claims,
         created_at = now()
       returning
          id"
 
     transaction(conn, fn conn ->
-      Enum.map(subscription_list, fn %{id: id, claims: claims, subscription_params: params = {schema, table, filters}} ->
-        case query(conn, sql, [publication, schema, table, id, claims, filters]) do
+      Enum.map(subscription_list, fn %{
```

Comment:
later we should extract this to a function as this is a bit cluttered but not important now